cmd/account/servicequotas/cmd.go

```diff
@@ -0,0 +1,25 @@
+package servicequotas
+
+import (
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+)
+
+// NewCmdServiceQuotas implements commands related to AWS service-quotas
+func NewCmdServiceQuotas(streams genericclioptions.IOStreams, flags *genericclioptions.ConfigFlags) *cobra.Command {
+	baseCmd := &cobra.Command{
+		Use:               "servicequotas",
```

Comment:
Cobra Command supports `Aliases`, consider adding an alias for servicequota (singular)